superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -593,6 +593,7 @@ export default function transformProps(
           extractForecastValuesFromTooltipParams(forecastValue);
 
         const keys = Object.keys(forecastValues);
+        let focusedRow = -1;
```

Comment:
nit: not a strong opinion, but as `focusedRow` already accepts `undefined`, I think it could be more explicit to set this to `undefined` here to clarify that it's in fact undefined.